src/utils/bot-utils.ts

```diff
@@ -29,7 +30,7 @@ export interface BotCommand {
    *
```

Comment:
Add to this description that an interaction should have a reply or else the user will get an "interaction failed" message when invoking a command.